webapp-django/crashstats/cron/management/commands/cronrun.py

```diff
@@ -166,8 +166,8 @@ def _run_one(self, job_spec, force=False, cmd_args=None):
                 )
 
                 # Send error to sentry, log it, and remember the failure
-                sentry_sdk.capture_error()
-                logger.error('error when running %s (%s): %s', cmd, run_time, single_line_tb)
+                sentry_sdk.capture_exception()
+                logger.warning('error when running %s (%s): %s', cmd, run_time, single_line_tb)
```

Comment:
Will the `cmd` and `run_time` end up in the report on Sentry? If not, we should add those as metadata.

Reply:
The local stack variables will show up in the stacktrace. Here's one where I added ``raise RuntimeError('boo')`` in ``_run_one``, line 151, you can see ``cmd`` and ``run_time`` in the ``vars`` section:

```json
{
  "level": "error",
  "exception": {
    "values": [
      {
        "type": "RuntimeError",
        "value": "boo",
        "stacktrace": {
          "frames": [
            {
              "filename": "crashstats/cron/management/commands/cronrun.py",
              "abs_path": "/app/webapp-django/crashstats/cron/management/commands/cronrun.py",
              "function": "_run_one",
              "module": "crashstats.cron.management.commands.cronrun",
              "lineno": 151,
              "pre_context": [
                "                    logger.info(\"running: %s %s %s\", cmd, cmd_args, cmd_kwargs)",
                "",
                "                    start_time = time.time()",
                "                    self._run_job(job_spec, *cmd_args, **cmd_kwargs)",
                "                    end_time = time.time()"
              ],
              "context_line": "                    raise RuntimeError(\"boo\")",
              "post_context": [
                "",
                "                    logger.info(\"successfully ran %s on %s\", cmd, run_time)",
                "                    last_success = run_time",
                "                    self._remember_success(cmd, last_success, end_time - start_time)",
                ""
              ],
              "vars": {
                "single_line_tb": "\"Traceback (most recent call last):\\n  File \"/app/webapp-django/crashstats/cron/management/commands/cronrun.py\", line 151, in _run_one\\n    raise RuntimeError(\"boo\")\\nRuntimeError: boo\\n\"",
                "end_time": "1558538754.7532594",
                "last_success": "datetime.datetime(2019, 5, 22, 15, 24, 54, 25764, tzinfo=<UTC>)",
                "cmd_kwargs": {},
                "run_time": "datetime.datetime(2019, 5, 22, 15, 25, 54, 745414, tzinfo=<UTC>)",
                "start_time": "1558538754.7459476",
                "exc_tb": "<traceback object at 0x7f095a931708>",
                "exc_value": "RuntimeError(\"boo\",)",
                "exc_type": "<class \"RuntimeError\">",
                "log_run": "True",
                "now": "datetime.datetime(2019, 5, 22, 15, 25, 54, 743066, tzinfo=<UTC>)",
                "seconds": "86400",
                "job": "<Job: <Job: crontest>>",
                "cmd": "\"crontest\"",
                "cmd_args": [],
                "force": "False",
                "job_spec": {
                  "cmd": "\"crontest\"",
                  "frequency": "\"1d\""
                },
                "self": "<crashstats.cron.management.commands.cronrun.Command object at 0x7f096161f828>"
              },
              "in_app": true
            }
          ]
        }
      }
    ]
  },
  "more": "omitted"
}
```

Reply:
I was thinking cmd is a lot like the crash id that we add to processor errors--it really gives us context to the exception.

For example, this is super helpful and easy to eyeball:

![image](https://user-images.githubusercontent.com/820826/58189217-0bfc6e80-7c88-11e9-9bb6-447c074c9227.png)

I'd love to have that for cronrun errors.

Reply:
Maybe a better way to fix this is to ditch the `sentry_sdk` call altogether and keep the logging line at `error` level?